wrapper/src/main/java/software/amazon/jdbc/plugin/failover/FailoverConnectionPluginFactory.java

```diff
@@ -25,6 +25,6 @@ public class FailoverConnectionPluginFactory implements ConnectionPluginFactory
 
   @Override
   public ConnectionPlugin getInstance(final PluginService pluginService, final Properties props) {
```

Comment:
Do we want to deprecate this method and add a new one that accepts service container and properties?

Reply:
I have a question about this, see my comment [here](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1371#discussion_r2064368233)